keystore/filesystem/server_keystore.go

```diff
@@ -909,6 +926,14 @@ func (store *KeyStore) DescribeKeyFile(fileInfo os.FileInfo) (*keystore.KeyDescr
 		}, nil
 	}
 
+	if lastKeyPart == "server" || lastKeyPart == "server.pub" || lastKeyPart == "translator" || lastKeyPart == "translator.pub" {
```

Comment:
Correct me if I am wrong. 

Here, you are checking that if fileInfo contains `.pub` suffix and you truncate it, right?
https://github.com/cossacklabs/acra/pull/510/files#diff-ca8937b8ce39cdc821c3d9b826aecd30d40cc5beb2aec26ea11497f39e13658fR847

So why are you checking scenarios that `lastKeyPart == "server.pub"` or `lastKeyPart == "translator.pub"`. Is it not enough to just check ` lastKeyPart == "server" || lastKeyPart == "translator"`

Reply:
See these lines above
```golang
components := strings.Split(fileInfo.Name(), "_") // note the separator
// ...
lastKeyPart := components[len(components)-1]
```
And checked while debugging that `lastKeyPart` really contains `server.pub` or `translator.pub` when corresponding files exist.